src/ncp/spinel.h

```diff
@@ -1891,6 +1891,40 @@ typedef enum
     SPINEL_PROP_NEST_LEGACY_LAST_NODE_JOINED
                                         = SPINEL_PROP_NEST__BEGIN + 2,
 
+    /// Thread network time.
+    /** Format: `Xc` - Read only
+     *
+     * Data per item is:
+     *
+     *  `X`: The Thread network time, in microseconds.
+     *  `c`: Time synchronization status.
+     *
+     */
+    SPINEL_PROP_NEST_THREAD_NETWORK_TIME
```

Comment:
How about adding the properties in `SPINEL_PROP_OPENTHREAD__BEGIN` section instead (since this is OpenThread provided feature)?